From 2f54ed4461cb8d361a9843cfc070b222d2269315 Mon Sep 17 00:00:00 2001
From: jgudec <jakov.gudec@gmail.com>
Date: Thu, 14 Jul 2022 08:23:51 +0100
Subject: [PATCH 11/11] Fix a CTS issue

---
 base.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/base.mk b/base.mk
index 2733322..f951d1e 100644
--- a/base.mk
+++ b/base.mk
@@ -61,7 +61,7 @@ PRODUCT_COPY_FILES += \
 	device/phh/treble/fixSPL/getSPL.arm:system/bin/getSPL
 
 PRODUCT_COPY_FILES += \
-	device/phh/treble/empty:system/phh/empty \
+	device/phh/treble/empty:system/phh/secure/empty \
 	device/phh/treble/phh-on-boot.sh:system/bin/phh-on-boot.sh
 
 PRODUCT_PACKAGES += \
-- 
2.17.1

